plugins/jobs/src/js/JobsToggleSchedule.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable no-unused-vars */
+import React from "react";
+/* eslint-enable no-unused-vars */
+
+import { graphqlObservable } from "data-service";
+import gql from "graphql-tag";
+
+import { Subject } from "rxjs/Subject";
+import "rxjs/add/operator/combineLatest";
+import "rxjs/add/operator/do";
+import "rxjs/observable/empty";
+
+import { jobsMenuQLSchema } from "./JobsMenuQLSchema";
+
+const enableJobMutation = gql`
+  mutation {
+    updateSchedule(jobId: $jobId, data: $data) {
+      jobId
+      data
+    }
+  }
+`;
+
+const enableJobGraphql = job => {
+  return graphqlObservable(enableJobMutation, jobsMenuQLSchema, job);
```

Comment:
For instance, if you had here:
```
updateSchedule(jobId: $jobId, data: $data, searchTerm: $search) {
}
```

The function should be called as:
```graphqlObservable(enableJobMutation, jobsMenuQLSchema, {jobId: 10, data: {}, search: "bananas"});```

Reply:
Yes that's what I have.

`job` is
```
{
jobId: String,
data: {...}
}
```